### ISSUE TITLE
Support the new ABI keywords

### ISSUE THREAD

**Issue by axic** [ethereum/browser-solidity#223]
```
[
  {
    "constant": false,
    "inputs": [],
    "name": "rsaverify",
    "output": [
      {
        "name": "",
        "type": "bool"
      }
    ],
    "payable": false,
    "type": "function"
  },
  {
    "inputs": [],
    "type": "constructor"
  },
  {
    "payable": false,
    "type": "fallback"
  }
]
```

UDapp only supports `function` and doesn't care about the new keywords.

Todo:
- [x] support the `constructor` keyword (right now it is created manually)
- [x] support the `fallback` keyword (right now it is created manually)
- [x] support the `payable` keyword and introduce an icon / colour for payable calls
- [x] register for the `compilerLoaded` event, because the current behaviour must be enabled for old compiler versions

**Comment by axic** [comment]
It may make more sense adding a filtering layer to the compiler: for "old" compiler versions, the interface would be extended with:
- a `constructor` entry
- a `fallback` entry
- `payable` keyword on every `constructor`, `fallback`, `function` entry

**Comment by axic** [comment]
Fixed by #331.
